.githooks/check-format.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo 'Checking formatting...'
+deepest_level=$(find . -name '*.dhall' | sed 's/[^\/]//g' | awk '{ print length }' | sort | tail -n 1)
+for depth in $(seq "${deepest_level}" -1 1) ; do
+  while IFS= read -r -d '' dhall_file ; do
```

Comment:
I am not sure what you are trying to achieve with the loop inside the loop 
Why not just doing?

```for i in $(find . -name 'package.dhall')
do
  dhall format --check --inplace $i
done